get-started/mcp/sdk-integrations/openai-agents.md

```diff
@@ -0,0 +1,326 @@
+mcp_token = getpass.getpass("Enter your MCP token: ")
+
+# Create kluster.ai client
+kluster_client = AsyncOpenAI(
+    base_url="https://api.kluster.ai/v1",
+    api_key=api_key
+)
+
+# Configure MCP server for verification
+mcp_server = MCPServerStreamableHttp(
+    params={
+        "url": "https://api.kluster.ai/v1/mcp",
+        "headers": {"Authorization": f"Bearer {mcp_token}"},
+        "timeout": 15,
+        "sse_read_timeout": 15
+    },
+    client_session_timeout_seconds=30  # Increase from default 5 seconds to 30 seconds
+)
+
+# Create agent with kluster Verify capabilities
+agent = Agent(
+    name="ReliableAgent",
+    instructions="Use kluster Verify for factual claims. Verify can detect hallucinations and validate information against real-time sources.",
+    model=OpenAIChatCompletionsModel(
+        model="deepseek-ai/DeepSeek-V3-0324",
+        openai_client=kluster_client
+    ),
+    mcp_servers=[mcp_server]
+)
+
+# Run the agent
+async def main():
+    await mcp_server.connect()
+    result = await Runner.run(agent, "Is the Earth flat?")
+    print(result.final_output)
+
+asyncio.run(main())
+```
+
+That's all you need to start with OpenAI Agents SDK and kluster Verify! Next, this guide will explore building an interactive chatbot that showcases kluster Verify's hallucination detection and real-time verification capabilities.
+
+!!! info "Model requirement"
+    Only models with **tool support** can use MCP verification. Filter models by "Tool Support" on the [platform models page](https://platform.kluster.ai/models){target=_blank} to see compatible options.
+
+!!! warning "Known async cleanup issue"
+    You may see error messages about "Exception ignored in atexit callback" when the script exits. This is a [known issue](https://github.com/modelcontextprotocol/python-sdk/issues/521){target=_blank} with MCP's asyncio cleanup that **does not affect functionality**. Your script will work correctly despite these messages.
+
+!!! info "Self-hosted option"
+    For self-hosted MCP, use `http://localhost:3001/stream` and your kluster.ai API key.
+
+## Enable MCP
+
+If you prefer, you can enable MCP and obtain your token via API calls instead of using the platform interface. This approach allows you to automate the token retrieval process and integrate it directly into your application setup workflow.
+
+```python
+import requests
+
+# Enable MCP
+response = requests.post(
+    "https://api.kluster.ai/v1/mcp/enable",
+    headers={"Authorization": f"Bearer {api_key}"}
+)
+
+# Get status with token
+status = requests.get(
+    "https://api.kluster.ai/v1/mcp/status", 
+    headers={"Authorization": f"Bearer {api_key}"}
+)
+mcp_token = status.json()["apiKey"]
+```
+
+## Build an interactive chatbot
+
+This example creates an interactive chatbot where you can ask questions and see kluster Verify validate the responses in real-time, demonstrating how verification enhances conversational AI.
```

Comment:
This is a pretty big code block you're dropping here. Nothing needs explanation?